data/config.dist.json

```diff
@@ -2,8 +2,8 @@
   "symbol-whitelist" : [
     "null", "true", "false",
     "static", "self", "parent",
-    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object",
-    "Composer\\InstalledVersions"
+    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object"
+    // add "Composer\\InstalledVersions" if composer-runtime-api:^2 reqired
```

Comment:
Hmm, no, suggesting that the end user has to do it doesn't seem right :thinking: